hoomd/md/TableAngleForceCompute.cc

```diff
@@ -9,9 +9,6 @@
     \brief Defines the TableAngleForceCompute class
 */
 
-#undef VT0
-#undef VT1
-
```

Comment:
Why were these removed?

Reply:
They are unnecessary. Nowhere does the code `#define VT0`  or `#define VT1`.